src/storage/kura.rs

```diff
@@ -309,68 +309,86 @@ impl Disk {
         Ok(model::Block::from(buffer))
     }
 
-    //TODO: implement reading all blocks
-    fn read_vec(&self) -> Vec<model::Block> {
-        Vec::new()
+    /// returns a sorted vector of blocks starting from 0 height to the top block
```

Comment:
Docs like a normal sentences should start from the capital letter and end with full stop.